### ISSUE TITLE
Help With Adjusting Hyperparameters 

### ISSUE THREAD

**Issue by elijahnavarro** [Unity-Technologies/ml-agents#1905]
I am currently building a tron-like game and all of the training I have performed has resulted in agents that are not performing as expected. The game consists of a 50x50 board and two agents. Each agent collects 2504 observations; 2500 for the state of the board at each step, 2 for this agent's position and direction, and 2 for the opponent's position and direction. Each agent can do nothing which will result in them going straight, they can turn left, and they can turn right. While they are constantly moving, mines are dropped behind them and death occurs when one player hits (1) a mine, (2) a wall, or (3) another player. On demand decision learning is being used. I have trained multiple times, the longest of which being 10 million steps. I thought that perhaps my hyperparameters were not optimized and was looking for help with it. Here is the current state of my configuration file.

```
default:
    trainer: ppo
    batch_size: 512
    beta: 1e-3
    buffer_size: 51200
    epsilon: 0.15
    gamma: 0.9
    hidden_units: 160
    lambd: 0.925
    learning_rate: 1e-5
    max_steps: 1e7
    memory_size: 512
    normalize: false
    num_epoch: 4
    num_layers: 2
    time_horizon: 960
    sequence_length: 64
    summary_freq: 1000
    use_recurrent: false
    use_curiosity: false
    curiosity_strength: 0.05
    curiosity_enc_size: 128
```
A little bit more on the game. I have a endListener that adds a rewards at the end of an episode. If this agent won, they get a reward of 0.001. If this agent lost, they get a reward of -1. If the round ended in a tie (both agents ran into each other), a reward of -0.5 is given. Also, is it okay for each agent (there are 2) to use the same learning brain and have only that one learning brain in the Academy's broadcast hub or should they have separate learning brains? Any help or guidance would be greatly appreciated!

**Comment by mattinjersey** [comment]
2500 is a lot of observations...might need a larger network for that which will take a long time to train. I wonder if there is a way to reduce the number of observations. Or maybe it would go better if you switch to image-based analysis rather than a fully-connected network.

**Comment by Eli-Candoris** [comment]
Yeah I was thinking that 2500 observations may be a lot as well but I was not sure. Switching to image-based analysis seems like it could be useful.

**Comment by elijahnavarro** [comment]
Yeah I was thinking that 2500 observations may be a lot as well but I was not sure. Switching to image-based analysis seems like it could be useful.

**Comment by ervteng** [comment]
Thanks for reaching out to us. Hopefully you were able to resolve your issue.  We are closing this due to inactivity, but if you need additional assistance, feel free to reopen the issue.
